packages/vite/src/node/plugins/dynamicImportPolyfill.ts

```diff
@@ -1,3 +1,23 @@
-throw new Error(
-  `Vite's dynamic import polyfill has been removed. Please install and import https://github.com/GoogleChromeLabs/dynamic-import-polyfill instead.`
-)
+import { ResolvedConfig } from '..'
+import { Plugin } from '../plugin'
+
+export const polyfillId = 'vite/dynamic-import-polyfill'
+
+export function dynamicImportPolyfillPlugin(config: ResolvedConfig): Plugin {
+  return {
+    name: 'vite:dynamic-import-polyfill',
+    resolveId(id) {
+      if (id === polyfillId) {
+        return id
+      }
+    },
+    load(id) {
+      if (id === polyfillId) {
+        config.logger.warn(
+          `\nVite's dynamic import polyfill has been removed, stop importing 'vite/dynamic-import-polyfill'`
```

Comment:
```suggestion
          `\n'vite/dynamic-import-polyfill' is no longer needed, refer to https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md#230-2021-05-10`
```

`stop importing` sounds a bit less friendly to me 😅. And kinda think we should link to the changelog or the issue about why we made this decision / how it affects.

Reply:
I like strong call to actions 👀